frontend/apps/reader/modules/readerfont.lua

```diff
@@ -486,7 +486,7 @@ This setting allows scaling all monospace fonts by this percentage so they can f
         text = _("Generate font test document"),
         callback = function()
             UIManager:show(ConfirmBox:new{
-                text = _("Would you like to generate an HTML document showing some sample text rendered with each available font?");
+                text = _("Would you like to generate an HTML document showing some sample text rendered with each available font?"),
```

Comment:
I suppose not at a glance, but this looks like a fix, not a minor stylistic nit?

Reply:
???

Reply:
I would prefer a separate fix commit to prevent a deceptive git blame.

Reply:
I don't think it has to be considered being a fix.
Semicolons, this one and all others, are not problematic, and are considered just like commas when in table definitions I think. So, it's all only cosmetics.
(I can speak, because a lot of these semicolons are mine ! :)

Reply:
Really? That's unexpected imo, but okay.

Reply:
Hurray for weird Lua syntactic tricks? :D